integration/container/stop_test.go

```diff
@@ -72,3 +113,18 @@ func containerIsStopped(ctx context.Context, client client.APIClient, containerI
 		}
 	}
 }
+
+func containerIsRunning(ctx context.Context, client client.APIClient, containerID string) func(log poll.LogT) poll.Result {
+	return func(log poll.LogT) poll.Result {
+		inspect, err := client.ContainerInspect(ctx, containerID)
+
+		switch {
+		case err != nil:
+			return poll.Error(err)
+		case inspect.State.Running:
+			return poll.Success()
+		default:
+			return poll.Continue("waiting for container to be running")
```

Comment:
If you include the  `inspect.State` in this message it will be included in the timeout message, which may help debugging.